core/sql/optimizer/BindRelExpr.cpp

```diff
@@ -10383,6 +10423,310 @@ NABoolean Insert::isUpsertThatNeedsMerge(NABoolean isAlignedRowFormat, NABoolean
      return FALSE;
 }
 
+#ifdef __ignore
```

Comment:
Put it in comments with a clear comment that we could try out a perf enhancement in the future.